ert3/engine/_run.py

```diff
@@ -79,11 +81,14 @@ def _transmitter_map_resources(
     futures: List[TransmitterCoroutine] = []
     for input_ in inputs:
         file_path = workspace.get_resources_dir() / input_.source_location
-        collection = ert.data.load_collection_from_file(
-            file_path,
-            input_.source_mime,
-            ensemble_size=ensemble_size,
-            is_directory=input_.source_is_directory,
+
+        collection = get_event_loop().run_until_complete(
+            ert.data.load_collection_from_file(
+                file_path,
+                input_.source_mime,
+                ensemble_size=ensemble_size,
+                is_directory=input_.source_is_directory,
+            )
```

Comment:
I am not sure how to asyncify this properly, possibly `transmit_record_collection` should await `load_collection_from_file` since its job seems to be to set up awaitables for later gatherings.